app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/PermissionWidget.java

```diff
@@ -23,7 +23,7 @@
 
 import java.net.URI;
 
-public class PermissionWidget extends UIWidget implements WidgetManagerDelegate.FocusChangeListener {
+public class PermissionWidget extends UIDialog implements WidgetManagerDelegate.FocusChangeListener {
```

Comment:
Is this a typo? Since UIDialog is derived from FocusChangeListener, does PermissionWidget need to be as well?